pkg/skaffold/color/formatter.go

```diff
@@ -62,13 +62,22 @@ func SetupColors(out io.Writer, defaultColor int, forceColors bool) {
 		37: White,
 		0:  None,
 	}[defaultColor]
+
+	if useColors {
+		return NewWriter(out)
+	}
+	return out
 }
 
 // Color can be used to format text so it can be printed to the terminal in color.
 type Color struct {
 	color *colors.Color
 }
 
+type colorableWriter struct {
```

Comment:
nit: you can do an embedded struct like:
```go
type colorableWriter struct {
	io.Writer
}
```
then you don't need to redefine `Write` method.

Reply:
Neat, golang always impresses me.